chapters.md

```diff
@@ -222,4 +78,236 @@ The next 4 following octets correspond to the [User Operation flags](http://dvd.
+      </ChapterAtom>
+      <ChapterAtom>
+        <ChapterUID>5</ChapterUID>
+        <ChapterTimeStart>00:10:33.000000000</ChapterTimeStart>
+        <ChapterTimeEnd>00:12:28.000000000</ChapterTimeEnd>
+        <ChapterDisplay>
+          <ChapterString>Baby wants to bleep (pt.3)</ChapterString>
+          <ChapterLanguage>eng</ChapterLanguage>
+        </ChapterDisplay>
+        <ChapterFlagHidden>0</ChapterFlagHidden>
+        <ChapterFlagEnabled>1</ChapterFlagEnabled>
+      </ChapterAtom>
+      <ChapterFlagHidden>0</ChapterFlagHidden>
+      <ChapterFlagEnabled>1</ChapterFlagEnabled>
+    </ChapterAtom>
+    <ChapterAtom>
+      <ChapterUID>6</ChapterUID>
+      <ChapterTimeStart>00:12:30.000000000</ChapterTimeStart>
+      <ChapterTimeEnd>00:19:38.000000000</ChapterTimeEnd>
+      <ChapterDisplay>
+        <ChapterString>Bleeper_O+2</ChapterString>
+        <ChapterLanguage>eng</ChapterLanguage>
+      </ChapterDisplay>
+      <ChapterFlagHidden>0</ChapterFlagHidden>
+      <ChapterFlagEnabled>1</ChapterFlagEnabled>
+    </ChapterAtom>
+    <ChapterAtom>
+      <ChapterUID>7</ChapterUID>
+      <ChapterTimeStart>00:19:40.000000000</ChapterTimeStart>
+      <ChapterTimeEnd>00:22:20.000000000</ChapterTimeEnd>
+      <ChapterDisplay>
+        <ChapterString>Baby wants to bleep (pt.4)</ChapterString>
+        <ChapterLanguage>eng</ChapterLanguage>
+      </ChapterDisplay>
+      <ChapterFlagHidden>0</ChapterFlagHidden>
+      <ChapterFlagEnabled>1</ChapterFlagEnabled>
+    </ChapterAtom>
+    <ChapterAtom>
+      <ChapterUID>8</ChapterUID>
+      <ChapterTimeStart>00:22:22.000000000</ChapterTimeStart>
+      <ChapterTimeEnd>00:25:18.000000000</ChapterTimeEnd>
+      <ChapterDisplay>
+        <ChapterString>Bleep to bleep</ChapterString>
+        <ChapterLanguage>eng</ChapterLanguage>
+      </ChapterDisplay>
+      <ChapterFlagHidden>0</ChapterFlagHidden>
+      <ChapterFlagEnabled>1</ChapterFlagEnabled>
+    </ChapterAtom>
+    <ChapterAtom>
+      <ChapterUID>9</ChapterUID>
+      <ChapterTimeStart>00:25:20.000000000</ChapterTimeStart>
+      <ChapterTimeEnd>00:33:35.000000000</ChapterTimeEnd>
+      <ChapterDisplay>
+        <ChapterString>Baby wants to bleep (k)</ChapterString>
+        <ChapterLanguage>eng</ChapterLanguage>
+      </ChapterDisplay>
+      <ChapterFlagHidden>0</ChapterFlagHidden>
+      <ChapterFlagEnabled>1</ChapterFlagEnabled>
+    </ChapterAtom>
+    <ChapterAtom>
+      <ChapterUID>10</ChapterUID>
+      <ChapterTimeStart>00:33:37.000000000</ChapterTimeStart>
+      <ChapterTimeEnd>00:44:28.000000000</ChapterTimeEnd>
+      <ChapterDisplay>
+        <ChapterString>Bleeper</ChapterString>
+        <ChapterLanguage>eng</ChapterLanguage>
+      </ChapterDisplay>
+      <ChapterFlagHidden>0</ChapterFlagHidden>
+      <ChapterFlagEnabled>1</ChapterFlagEnabled>
+    </ChapterAtom>
+    <EditionFlagDefault>0</EditionFlagDefault>
+    <EditionFlagHidden>0</EditionFlagHidden>
+  </EditionEntry>
+</Chapters>
+```
```

Comment:
Minimum chapter.xml sample

I  have noticed that some program's output of a chapter.xml is not Matroska conform. 
(Missing UID's or Flags)
As I started my capterEditor project, I  would have been glad to find such info.

Here is a minium sample in xml:
<Chapters>
  <EditionEntry>
    <EditionFlagHidden>0</EditionFlagHidden>
    <EditionFlagDefault>0</EditionFlagDefault>
    <ChapterAtom>
      <ChapterUID>1</ChapterUID>
      <ChapterTimeStart>00:00:00.000000000</ChapterTimeStart>
      <ChapterFlagHidden>0</ChapterFlagHidden>
      <ChapterFlagEnabled>1</ChapterFlagEnabled>
    </ChapterAtom>
  </EditionEntry>
</Chapters>

And then we could have an "absolute minimum", because the Flags(Hidden, Enabled) can use the default values. 
<Chapters>
  <EditionEntry>
    <ChapterAtom>
      <ChapterUID>1</ChapterUID>
      <ChapterTimeStart>00:00:00.000000000</ChapterTimeStart>
    </ChapterAtom>
  </EditionEntry>
</Chapters>